6.crud-php/Controllers/projects.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once('controller.php');
+
+
+class projects extends controller
+{
+
+    public $model_cls = "projects";
+    protected $user_id;
+    protected $account_id;
```

Comment:
you don't need to write these fields on each controller. 
the fact that this class extends class controller make it automatically have account & user id